force-app/tdtm/classes/TDTM_DefaultConfig.cls

```diff
@@ -191,7 +191,7 @@ public without sharing class TDTM_DefaultConfig {
 
         handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
               Class__c = 'RD2_RecurringDonations_TDTM', Load_Order__c = 2, Object__c = 'npe03__Recurring_Donation__c',
-              Trigger_Action__c = 'BeforeDelete;BeforeInsert;BeforeUpdate;AfterInsert;AfterUpdate'));
+              Trigger_Action__c = 'BeforeDelete;BeforeInsert;BeforeUpdate;AfterInsert;AfterUpdate;AfterDelete;AfterUndelete'));
```

Comment:
Not sure if this is change will actually update customer org to enable AfterDelete and AfterUndelete for `RD2_RecurringDonations_TDTM`.

Reply:
It seems like STG_InstallScript will run a TDTM Handler update during the package install/upgrade.

Reply:
Great find! I should have looked closer at this 🙃